.cicd/docker/.helpers-v33

```diff
@@ -0,0 +1,8 @@
+function fold-execute() {
```

Comment:
We don't need the code folding. This is unnecessary complexity. This can be the same as above, and should not be duplicated here IMO:
```
function execute()
{
    echo "\$ $@"
    "$@"
}
```

Reply:
I do; It makes troubleshooting failures way easier.